### ISSUE TITLE
network.test() method needs a-fixin

### ISSUE THREAD

**Issue by jmarshallgit** [Calysto/conx#126]
test() method seems to evaluate the training set by default. also prints tolerance even if output layer uses softmax.

**Comment by dsblank** [comment]
Fixed.
